code/modules/shuttle/shuttle.dm

```diff
@@ -714,6 +714,8 @@
 		possible_destinations = C.possible_destinations
 		shuttleId = C.shuttleId
 
+/obj/machinery/computer/shuttle/Initialize()
```

Comment:
The computer used New(), whereas the docking controller used Initialize(). As such, the computer would look for the docking controller before the docking controller had registered itself with the shuttle controller.

Reply:
Does this solve 

`## WARNING: couldn't find shuttle with id: steel_rain in code/controllers/subsystem/shuttles.dm at line 87 src: Shuttle usr: `

Warning?

Reply:
It does, yes.

Reply:
Oof, I hate to sound like a butt, but this is just becuase I just spotted this in my own testing, yesterday:

make this `Initialize(mapload)`.

I was having issues with my ant PR when I was first coding it; some things wouldn't function properly even *with* `Initialize`, but when it was `Initialize(mapload)` the problem ceased.

While this will work *now* without the `mapload`, heaven forbid we make a change to computers, machinery, or even atoms that depends on it, and we're left wondering "why doesn't this work now?"

Reply:
Done.